get_started_with_membrane/06_flow_control.md

```diff
@@ -0,0 +1,41 @@
+# Flow Control
+
+Once we have linked the elements together, there comes a moment when they can start sending buffers through the pads. However, how to control the flow of buffers between elements?
+
+Membrane solves this problem using its own backpressure mechanism and demands.
+
+## Types of `flow_control`
+
+When defining a pad od an element, you can pass one of 3 values in the `flow_control` field: `auto`, `push`, or `manual`. They determine how the backpressure mechanism works on the pad they relate to.
+
+- input pad with `flow_control: :push` - an element with such a pad must always be ready to receive a new buffer on that pad. The output pad linked with such a input pad must also have `flow control` set to `:push`.
+
+- output pad with `flow_control: :push` - the element decides for itself when to send how many buffers via a given pad.
+
+- input pad with `flow_control: :manual` - the element directly, using the `demand` action, determines how much data it wants to receive from this pad. Membrane guarantees that more data will never come to the input pad than was demanded..
+
+- output pad with `flow_control: :manual` - the element gets the information, through the `handle_demand` callback, how much data a specific output pad needs. An element having an output pad with `:manual` `flow control` is obligated to satisfy the demand on it.
+
+- input pad with `flow_control: :auto` - since `v1.0.0-rc1` the default `flow control` value. Unlike in manual demands, the element must not directly specify the demand value on the input pad to receive buffers on it. In `:auto` `flow control`, the framework will manage the demand value itself. In the case of Sources and Endpoints, it will make sure that the demand on the input pad is positive as long as the number of buffers waiting for processing in the mailbox is not too large. In the case of filters, Membrane will take care of a positive demand on under one additional condition - the demand value on all output pads with `flow_control: :auto` of this element must also be positive.
```

Comment:
I wouldn't mention the new defaults until we have 1.0

Reply:
```suggestion
- input pad with `flow_control: :auto` - since `v1.0.0-rc1` the default `flow control` value. Unlike in manual demands, the element doesn't have to directly specify the demand value on the input pad to receive buffers on it. In `:auto` `flow control`, the framework will manage the demand value itself. In the case of Sources and Endpoints, it will make sure that the demand on the input pad is positive as long as the number of buffers waiting for processing in the mailbox is not too large. In the case of filters, Membrane will take care of a positive demand on under one additional condition - the demand value on all output pads with `flow_control: :auto` of this element must also be positive.
```